read_fan_speed.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
This should be #!/usr/bin/env python3. I goofed on this and couldn't remember the sequence. That's what's meant here. I'm going to kill this version and remake it.